### DISCUSSION TITLE
demo/%E5%89%8D%E7%AB%AF/page

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#707] [Announcements]
# demo/%E5%89%8D%E7%AB%AF/page

nvm是一个node的版本管理工具，可以简单操作node版本 (https://so.csdn.net/so/search?q=node版本&spm=1001.2101.3001.7020)的切换、安装、查看。。。等等，与npm不同的是，npm是依赖包的管理工具。 下载路径: https://github.com/coreybutler/nvm-win...

http://localhost:8080/demo/%E5%89%8D%E7%AB%AF/page.html

**Comment by zqtop** [comment]
房贷首付看来是
